src/common/hooks/useVersionHistoryFormatters/useVersionHistoryFormatters.js

```diff
@@ -0,0 +1,42 @@
+/* Developed collaboratively using AI (Chat GPT) */
+
+import { getFullName } from '@folio/stripes/util';
```

Comment:
Please take a look at the import order rules https://folio-org.atlassian.net/wiki/spaces/FOLIJET/pages/1377187/Code+style+rules+not+covered+by+eslint

Reply:
It is fixed now?